### PR TITLE
Added German Translation

### PR DESCRIPTION
**Pull request by ManuGithubSteam**
Hi

This is my first Github PR. Sorry for the messup with the ~ files.

Hope to find it useful.

Have a nice week
Manuel

### CODE REVIEW

**Comment by ManuGithubSteam** [comment]
Hmm
I don't know what it wrong here. Anyway as described in: https://github.com/dularion/streama/issues/246 i can not compile a grails3 branch.

**Comment by dularion** [comment]
yup sure :) 
thanks!

**Comment by dularion** [comment]
Manually merged! 
Thank you for the contribution! 
One note though, I am not sure I like the word "Tafelansicht", cant think of anything better either at the moment though.

File: grails-app/assets/javascripts/streama/translations/DE_de.js [hunk 1]
```diff
@@ -3,7 +3,137 @@
...
+				TITLE: 'Wie kann ich Untertitel zu Videos hinzufügen?',
+				TEXT: "Du kannst Untetertitel in der Videos Infromationsseite unter Inhalte verwalten. Du kannst dort Untetitel durch Drag and Drop hinzufügen." +
+				"Die Untertitel müssen nicht mehr konvertiert werden."
+			},
+			INVITE_USERS: {
+				TITLE: 'Wie kann ich Freunde einladen um meine gehosteten Videos an zu schauen?',
+				TEXT:"Du kannst deine Videos auf Streama teilen, indem du deine Frende einlädst. Gehe zum Benutzer Menu und klicke auf den Invite Button. Fülle das Formuular aus" +
+				" und wähle die Rollen. Benutzer mit der Rolle Admin können Benutzer und Einstellungen ändern. Benutzer mit der Rolle role Content Manager können Inhalte verwalten." +
+				" Deine Freunde werden über Email benachrichtigt, das du Sie eingeladen hast. 
+				Du kannst auch Video-Sessions teilen indem du auf den Share Button des Videoplaers drückst und die Session URL teilst."
```
Comment (bastilimbach) [inline grails-app/assets/javascripts/streama/translations/DE_de.js]: You could change "Video-Sessions" to "Videositzungen" and "Session" to "Sitzung" for consistency. Also "Videoplayer" is misspelled.

File: grails-app/assets/javascripts/streama/translations/DE_de.js [hunk 2]
```diff
@@ -3,7 +3,137 @@
...
+			},
+			BASE_URL: {
+				TITLE: "Was ist die basis URL und wie sollte Sie konfiguriert werden?",
+				TEXT: "Die base URL wird für Email Einladungen benutzt."
+			},
+			NOTIFICATIONS: {
+				TITLE: "Was sind Benachrichtigungen?",
+				TEXT: "Du kannst deine eingeladenen Freunde über hochgeladene Videos mit Benachrichtigungen informieren. Du kannst diese zur Benachrichtigungswarteschalnge hinzufügen." +
```
Comment (bastilimbach) [inline grails-app/assets/javascripts/streama/translations/DE_de.js]: Misspelling and I would change the sentence "Du kannst diese zur Benachrichtigungswarteschalnge hinzufügen. [...]" to "Du kannst diese zur Benachrichtigungswarteschlange hinzufügen, indem du den Benachrichtigungsknopf in der Informationsseite [...]"

File: grails-app/assets/javascripts/streama/translations/DE_de.js [hunk 3]
```diff
@@ -3,7 +3,137 @@
...
+				" Benachrichtigung Knopf in der Video Informationsseite drücken und im Benachrichtungsmenu auf Senden klicken"
+			},
+			VIDEO_PLAYER_SHORTCUTS: {
+				TITLE: "Hat der Player Kürzeltasten?",
```
Comment (bastilimbach) [inline grails-app/assets/javascripts/streama/translations/DE_de.js]: Would change "Kürzeltasten" to "Tastaturkurzbefehle"

**Comment by dularion** [comment]
Since these are many small changes, I'd appreciate another PR for this. Thanks for your help on this! Please be mindful to do a fresh fork or pull from the current status of the grails3 branch

**Comment by bastilimbach** [comment]
Alright. I'm on it!

**Comment by dularion** [comment]
Love the enthusiasm! :)

File: grails-app/assets/javascripts/streama/translations/DE_de.js [hunk 4]
```diff
@@ -3,7 +3,137 @@
...
+		},
+
+		SORT_OPTIONS: {
+			AZ: 'A-Z',
+			ZA: 'Z-A',
+			NEWEST_ADDED: 'Zuletzt hinzugefügt',
+			OLDEST_ADDED: 'Zuerst hinzugefügt',
+			NEWEST_RELEASED: 'Neuste Veröffentlichung',
+			OLDEST_RELEASED: 'Älteste Veröffentlichung',
+			NEWEST_AIRED: 'Zuletzt angeschaut',
```
Comment (bastilimbach) [inline grails-app/assets/javascripts/streama/translations/DE_de.js]: @dularion Could you tell me if the variable NEWEST_AIRED is used to display "Last watched" or "Last aired"?

Comment (dularion) [inline grails-app/assets/javascripts/streama/translations/DE_de.js]: NEWEST_AIRED: 'Most Recently Aired'

Also, feel free to check the original here:  https://github.com/ManuGithubSteam/streama/blob/4bfc4e011b3b196e2ba41f1af2867ae8a1a06d01/grails-app/assets/javascripts/streama/translations/EN_us.js

**Comment by ManuGithubSteam** [comment]
Thanks for the help on this! I was planing on refining the translation when i get my grails3 branch to build but that is even better to help together :-)

**Pull request closed without merging**